src/components/header.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
Rename these files to `Footer.tsx`, `Header.tsx` 
<img width="329" alt="image" src="https://github.com/user-attachments/assets/e3e746f7-14e9-4230-a904-ff91bcad8aa7" />